soc/x86/intel_ish/Kconfig

```diff
@@ -13,3 +13,4 @@ config SOC_FAMILY_INTEL_ISH
 	select CPU_HAS_FPU
 	select INTEL_HAL
 	select HAS_PM
+	select HAS_COVERAGE_SUPPORT
```

Comment:
not soc thing, grep shows it should be selected in boards/x86/intel_ish/Kconfig.board.

Reply:
+1 on moving to the board Kconfig.

Reply:
I think it's fine here, it has already been forgotten for a new board, and it has more to do with the arch than the board.